libraries/botframework-streaming/src/namedPipe/namedPipeServer.ts

```diff
@@ -20,48 +20,39 @@ import { createNodeServer } from '../utilities/createNodeServer';
 export class NamedPipeServer implements IStreamingTransportServer {
     private _outgoingServer: INodeServer;
     private _incomingServer: INodeServer;
-    private readonly _baseName: string;
-    private readonly _requestHandler: RequestHandler;
-    private readonly _sender: PayloadSender;
-    private readonly _receiver: PayloadReceiver;
-    private readonly _requestManager: RequestManager;
+
+    private readonly _sender = new PayloadSender();
+    private readonly _receiver = new PayloadReceiver();
     private readonly _protocolAdapter: ProtocolAdapter;
-    private readonly _autoReconnect: boolean;
-    private _isDisconnecting: boolean;
 
     /**
      * Creates a new instance of the [NamedPipeServer](xref:botframework-streaming.NamedPipeServer) class.
      *
      * @param baseName The named pipe to connect to.
      * @param requestHandler Optional [RequestHandler](xref:botframework-streaming.RequestHandler) to process incoming messages received by this client.
-     * @param autoReconnect Optional setting to determine if the client should attempt to reconnect automatically on disconnection events. Defaults to true.
+     * @param autoReconnect Optional flag to indicate that server should reconnect instead of closing

```

Comment:
Do we have a convention for deprecating arguments? I think pre-fixing the `@param` description with `DEPRECATED:` (or similar) would be nice so that developers can avoid the `console.warn` prematurely through intellisense.

Reply:
It seems [JSDoc does not support this](https://stackoverflow.com/questions/29763919/jsdoc-param-together-with-deprecated), so I'll opt for including `Deprecated: ` in the description. Good call!